confluence/confluence-v1.asciidoc

```diff
@@ -1,77 +1,99 @@
 
-* spaces 
-* pages (such as Wiki pages), 
-* blog posts,  
-* pages and blog posts comments (indexed as individual solr-docs)
+The list of data the REST connector crawls using the Confluence REST configuration is:
 
-Authentication
-~~~~~~~~~~~~~~~
-Support Basic Authentication with username (atlassian account) and API Token
+* Spaces 
+* Pages such as Wiki pages 
+* Blog posts  
+* Pages and blog posts comments that are indexed as individual solr-docs
 
-1. For more info see Documentation here - https://developer.atlassian.com/cloud/confluence/basic-auth-for-rest-apis/ 
 
-2. Check here to create new Token API https://id.atlassian.com/manage/api-tokens
+*Authentication methods*
+------------------------
 
+The Confluence REST configuration supports:
 
-Supported Crawl
-~~~~~~~~~~~~~~~
-* Full Crawl supported
-* Re-Crawl supported - `rely on the strayContentDeletion`
+* Basic Authentication using the username and password from an Atlassian account. For more information, see https://developer.atlassian.com/cloud/confluence/basic-auth-for-rest-apis/.
 
-Pagination
-~~~~~~~~~~~
-This recipe is configured to used pagination by batch size. The variable `${LW_INDEX_START}` is set to zero be the atlassian pagnation is zero index base.The `${LW_BATCH_SIZE}` is set to `20` but can be adjusted to any value.
-The pagination stop condition is set to `results` response object set `[]`. When this condition is meet the pagination will stop.
+* API Token. For information about how to create a new API token, see https://id.atlassian.com/manage/api-tokens.
 
-Rest Connector Variables used
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-* ${LW_BATCH_SIZE} - This variable is used to set the `limit` query parameter which controls the number results that are returned in the response for both `blogpost` and `pages`
 
-* ${LW_INDEX_START} - This variable is used to set the `start` query parameter which is used to traverse the pagination. Confluence pagination is 0 index based. This means the first page number is 0.
+*Supported crawl options*
+-------------------------
 
-* ${LW_PARENT_DATA_KEY} - This variable is uesd to access a value from the response from `main request`to used in a `additional request`. Confluence use case this variable added to URL to proform a GET request, to get comments for blogpost and pages
+The Confluence REST configuration supports the following crawl options:
 
+* Full crawl
 
-Endpoints to configure with Rest-Connector
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-The table below consists of a list of endpoints and descriptions of what each endpoint retrieves.
+* Recrawl that uses the `rely on the strayContentDeletion` parameter
+
+
+*Pagination information*
+------------------------
+
+The `confluence-v1.json` file is also referred to as a recipe. This recipe is configured to use pagination by batch size. 
```

Comment:
The `confluence-v1.json` is already referenced in the first section. I think only this is enough -> This recipe is configured to use pagination by batch size.

Reply:
Changed.